src/app.ts

```diff
@@ -0,0 +1,11 @@
+import bodyParser from 'body-parser'; 
+import helmet from 'helmet';
+import close,{ Express } from 'express';
+import router from '../routes/routes';
+const server = router
```

Comment:
Why? isn't `router` a good enough name?

Reply:
The fact is that maybe router should be an express router, and server an express server